queries/css/indents.scm

```diff
@@ -0,0 +1,9 @@
+[
+  (block)
+  (declaration)
+] @indent
+
+[
+  "}"
+] @branch
```

Comment:
shouldn't `{` also be a branch ?
```css
.some-class
{
  |# you want the cursor here
}

Reply:
although this shouldn't be happening too often ^^

Reply:
I've been trying to add `"{"` to `@branch` but it's not works. :(

Reply:
maybe its not a node, we'll merge this as it is and iterate if we find unwanted behaviors.